tests/test_download.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+from unittest import TestCase
+
+from morningstar import good_morning as gm
```

Comment:
Yes, it works.  If you install the library from my pull request, you should be able to run through the example.  I added this folder in the last pull request a month ago.